frontend/src/admin/components/dropdown-menu/dropdown-menu.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../../styles/main";
+
+.c-dropdown-menu {
+  position: relative;
+
+  button {
```

Comment:
This targets all buttons below in the dropdown menu. Is that intended behaviour or is it a specific button/ set of buttons that should be targeted?